src/Utils/GitHubUtils.ts

```diff
@@ -60,3 +65,15 @@ export function toContentUri(owner: string, repo: string, branch: string, path:
 export function toRawContentUri(owner: string, repo: string, branch: string, path: string): string {
   return `https://raw.githubusercontent.com/${owner}/${repo}/${branch}/${path}`;
 }
+
+export function GetGithubClientId(): string {
+  const junoEndpoint = userContext.features.junoEndpoint ?? configContext.JUNO_ENDPOINT;
```

Comment:
Is this needed? I thought we already update the values in configContext somewhere whenever feature flags are used. Can you confirm?

Reply:
we dont update config context itself, we do the exact same thing in JunoClient's getJunoEndpoint